### ISSUE TITLE
Cannot select last member of API documentation

### ISSUE THREAD

**Issue by peterennis** [dotnet/docfx#560]
Keyboard navigation does not work in the list.
Same for IE11 and Chrome.
![capture52](https://cloud.githubusercontent.com/assets/140737/17348447/0191773a-58cb-11e6-99db-a1d66d0a3904.PNG)
